src/AlignHorizontal/AlignHorizontal.tsx

```diff
@@ -1,23 +1,56 @@
 import * as React from "react";
+
 import {
   FormatAlignCenter,
   FormatAlignJustify,
   FormatAlignLeft,
   FormatAlignRight
 } from "@mui/icons-material";
 import { ToggleButton, ToggleButtonGroup } from "@mui/material";
-import PropTypes from "prop-types";
+
+export interface AlignHorizontalProps {
+  /**
+   * If true, the component is disabled.
+   */
+  disabled?: boolean;
+  /**
+   * Callback fired when the value changes.
+   *
+   * **Signature**
+   * ```
+   * function(event: React.MouseEvent<HTMLElement>, value: any) => void
+   * ```
+   * event: The event source of the callback.
+   * value: The value of the selected button.
+   */
+  onChange: (
+    event: React.MouseEvent<HTMLElement, MouseEvent>,
+    value: "left" | "center" | "right" | "justify" | null
+  ) => void;
+  /**
+   * The orientation of the toggle button group.
+   */
+  orientation?: "horizontal" | "vertical";
+  /**
+   * The size of the component.
+   */
+  size?: "small" | "medium" | "large";
+  /**
+   * The value of the selected button.
+   */
+  value?: "left" | "center" | "right" | "justify" | null;
+}
```

Comment:
I think we agreed to follow the MUI standard and always have the types defined in a .types file alongside the component, rather than inside the component.

e.g. from MUI

![image](https://user-images.githubusercontent.com/8976377/232844896-4f5016aa-a067-48f4-b6cd-c62f573437f2.png)

Reply:
Would we also need to export these types so we can import them in an app? Is there a need for that? Only if we are extending I guess?

Reply:
I think its probably good practice. No harm in exporting the types.

Reply:
> `npm run storybook` fails to start for me.
> 
> ![image](https://user-images.githubusercontent.com/8976377/232846413-8a04d5b6-8e32-4b89-b0be-5d925d9c443a.png)

Resolved in https://github.com/IPG-Automotive-UK/react-ui/pull/575#issuecomment-1513484652

Looks like we're not ready for typescript 5 yet, understandable since we're using old storybook, and typescript 5 is a month or two old. We'll need to upgrade to storybook 7 at some point (I got it semi working but one for another time).

Reply:
Resolved in 48a028575620aa4dd241d8f24b70c35b6356f130